modules/dnn/test/test_backends.cpp

```diff
@@ -545,7 +545,7 @@ TEST_P(DNNTestNetwork, FastNeuralStyle_eccv16)
     Mat img = imread(findDataFile("dnn/googlenet_1.png"));
     Mat inp = blobFromImage(img, 1.0, Size(320, 240), Scalar(103.939, 116.779, 123.68), false, false);
     // Output image has values in range [-143.526, 148.539].
-    float l1 = 4e-5, lInf = 2e-3;
+    float l1 = 5e-5, lInf = 2e-3;
```

Comment:
Please use x2-x5 values for test tolerance checks.

4e-5 => 1e-4 instead of 5e-5

---

Below:

1e-5 => 1e-4 instead of 1.01e-5